sunpy/net/jsoc/jsoc.py

```diff
@@ -58,7 +60,16 @@ def __len__(self):
             return len(self.table)
 
     def __getitem__(self, item):
-        return type(self)(self.table[item])
+        if isinstance(item, int):
+            item = slice(item, item + 1)
+        ret = type(self)(self.table[item])
+        ret.query_args = self.query_args
+        ret.requests = self.requests
```

Comment:
If these are `None` what happens?

Reply:
`__getitem__` is just used to get the slice of the response. All these lines preserve the properties of the original response, and so if they are originally `None` they can be safely passed on as `None`

Reply:
Fair enough.